### PR TITLE
Fix exception when process description field in the view for Python 3

### LINKED ISSUE #26: Py3: Exception when process description in the view
```python
Traceback (innermost last):
  Module ZPublisher.WSGIPublisher, line 155, in transaction_pubevents
  Module ZPublisher.WSGIPublisher, line 337, in publish_module
  Module ZPublisher.WSGIPublisher, line 255, in publish
  Module ZPublisher.mapply, line 85, in mapply
  Module ZPublisher.WSGIPublisher, line 61, in call_object
  Module zope.browserpage.simpleviewclass, line 41, in __call__
  Module Products.Five.browser.pagetemplatefile, line 126, in __call__
  Module Products.Five.browser.pagetemplatefile, line 61, in __call__
  Module zope.pagetemplate.pagetemplate, line 135, in pt_render
  Module Products.PageTemplates.engine, line 88, in __call__
  Module z3c.pt.pagetemplate, line 173, in render
  Module chameleon.zpt.template, line 306, in render
  Module chameleon.template, line 209, in render
  Module chameleon.utils, line 75, in raise_with_traceback
  Module chameleon.template, line 187, in render
  Module e89c160517ca26242297464f3b57bfc8, line 934, in render
  Module 7d802565c24d87384e7395ea91cc1cab, line 684, in render_master
  Module 7d802565c24d87384e7395ea91cc1cab, line 1274, in render_content
  Module e89c160517ca26242297464f3b57bfc8, line 924, in __fill_main
  Module e89c160517ca26242297464f3b57bfc8, line 640, in render_main
  Module kitconcept.glossary.browser.views, line 189, in result_features
TypeError: a bytes-like object is required, not 'str'

 - Expression: "python:result_features(result)"
 - Filename:   ... ry/src/kitconcept/glossary/browser/templates/glossary.pt
 - Location:   (line 87: col 44)
 - Source:     ... ry define="result python:result_features(result);
                                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 - Expression: "here/main_template/macros/master"
 - Filename:   ... ry/src/kitconcept/glossary/browser/templates/glossary.pt
 - Location:   (line 3: col 23)
 - Source:     metal:use-macro="here/main_template/macros/master"
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 - Arguments:  template: <ViewPageTemplateFile - at 0x111489710>
               options: {...} (0)
               args: <tuple - at 0x1065d9050>
               nothing: <NoneType - at 0x106472d58>
               modules: <_SecureModuleImporter - at 0x10852d8d0>
               request: <WSGIRequest - at 0x10ec06d10>
               view: <SimpleViewClass from /Users/rodrigo/.projects/kitconcept/kitconcept.glossary/src/kitconcept/glossary/browser/templates/glossary.pt view at 0x1115ebfd0>
               context: <ImplicitAcquisitionWrapper glossary at 0x111922640>
               views: <ViewMapper - at 0x1118a9090>
               here: <ImplicitAcquisitionWrapper glossary at 0x111922640>
               container: <ImplicitAcquisitionWrapper glossary at 0x111922640>
               root: <ImplicitAcquisitionWrapper  at 0x10c8e7910>
               traverse_subpath: <list - at 0x11188c370>
               user: <ImplicitAcquisitionWrapper - at 0x10cf81c80>
               default: <object - at 0x1075aaf00>
               repeat: {...} (0)
               loop: {...} (3)
               wrapped_repeat: <SafeMapping - at 0x10d01ef00>
               target_language: <NoneType - at 0x106472d58>
               translate: <function translate at 0x10dda38c0>
               macroname: master
```

### PR DESCRIPTION
**Pull request by rodfersou**
closes #26

### CODE REVIEW

File: src/kitconcept/glossary/browser/views.py [hunk 1]
```diff
@@ -175,7 +176,8 @@ def truncateDescription(self, text):
             text = text.strip()
             text = u"{0}{1}".format(text, ellipsis)
 
-        text = text.encode("utf-8", "replace")
```
Comment (tisto) [inline src/kitconcept/glossary/browser/views.py:178]: @rodfersou shouldn't we use encode here that works for both py2 and py3?

Comment (rodfersou) [reply src/kitconcept/glossary/browser/views.py:178]: the encode works, but later we try to do a replace that don't works.

https://github.com/kitconcept/kitconcept.glossary/blob/679a41529325dede45064657d7d67b42854ce4a4/src/kitconcept/glossary/browser/views.py#L189

I'll add a test to show better the problem after make the buildout green again on master.

Comment (rodfersou) [reply src/kitconcept/glossary/browser/views.py:178]: @tisto here I prove that it is broken for Python 3:
https://travis-ci.org/github/kitconcept/kitconcept.glossary/jobs/673332140

**Comment by rodfersou** [comment]
it is fixed like this, will merge.

File: src/kitconcept/glossary/tests/test_views.py [hunk 2]
```diff
@@ -68,3 +68,9 @@ class GlossaryViewTestCase(BaseViewTestCase):
     def setUp(self):
         super(GlossaryViewTestCase, self).setUp()
         self.view = api.content.get_view(u"view", self.g1, self.request)
+
+    def test_view(self):
+        try:
```
Comment (tisto) [inline src/kitconcept/glossary/tests/test_views.py:73]: @rodfersou what are you doing here? A try/except in a test?

Comment (rodfersou) [reply src/kitconcept/glossary/tests/test_views.py:73]: assert that not raises an exception..

Comment (tisto) [reply src/kitconcept/glossary/tests/test_views.py:73]: @rodfersou this does not make any sense to me TBH. You are replacing an exception that is thrown with a test failure.

Comment (rodfersou) [reply src/kitconcept/glossary/tests/test_views.py:73]: @tisto learned something new, will fix, thanks.

**Pull request merged**